File: DSA Lab/(Class 1) 26_7_2021/Q5.cpp
#include<iostream>
using namespace std;

int main()
{
    int sum=0,n;
    cout<<"Enter the number till N =  ";

    cin>>n;
    for(int i=1;i<=n;i++)
    sum+=5*i;
    cout<<sum;
    return 0;
}

File: TLL labwork/lab2/q5.py
print("enter number= ")
a=int(input())

x=a%10
x=x*100
a=a/10
a=a/10
a=int(a)
a=a-(a*100)
a=a+x;
y=a%10

print(x)
print(y)
print('ANS=',x+y)
